protocols/discord.py

```diff
@@ -603,9 +643,9 @@ def _message_builder(self):
                 current_sender = current_channel_senders.get(channel, None)
 
                 # We'll enable this when we work on webhook support again...
-                #if current_sender != message['sender']:
-                #    self.flush(channel, joined_messages[channel])
-                #    joined_messages[channel] = message
+                if current_sender != message['sender']:
+                   self.flush(channel, joined_messages[channel])
+                   joined_messages[channel] = message
```

Comment:
4 spaces please - also I think this should check the webhooks option too, since text doesn't need to be batched into separate messages if webhooks are turned off?

Reply:
If webhooks are enabled, then `current_sender` will always be the clientbot, so this check would never pass.

Reply:
I see!